repowiz-application/src/main/java/life/qbic/repowiz/application/view/RepoWizView.java

```diff
@@ -95,11 +111,16 @@ public void setUserAnswer(UserAnswer answer) {
         this.answer = answer;
     }
 
-    private String mapToString(HashMap<Integer, String> map){
+    /**
+     * Creates a String from a map whereas each key value is printed next its value in a single line
+     * @param map with Integers as keys and Strings as values
+     * @return text with information of the map
+     */
+    private String mapToString(HashMap<Integer, String> map) {
```

Comment:
```suggestion
    private String mapToString(Map<Integer, String> map) {
```
Using the interface here is better. You only use interface methods so this should be fine.

Reply:
done